src/cascadia/WindowsTerminal/TrayIcon.cpp

```diff
@@ -20,9 +22,41 @@ TrayIcon::TrayIcon(const HWND owningHwnd) :
 
 TrayIcon::~TrayIcon()
 {
+    DestroyWindow(_trayIconWndProc);
     RemoveIconFromTray();
 }
 
+void TrayIcon::CreateWindowProcess()
+{
+    WNDCLASSW wc{};
+    wc.hCursor = LoadCursor(nullptr, IDC_ARROW);
+    wc.hInstance = reinterpret_cast<HINSTANCE>(&__ImageBase);

```

Comment:
Huh, i suspect this got copied from some other Cascadia stuff. Surprised it wasn't using a wil helper!